### DISCUSSION TITLE
А можно как-нибудь так сделать, чтоб пользователь видел только таблицу внутри цикла, а таблицу цикла не видел?

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#435] [Как это сделать?] [answered]
А можно как-нибудь так сделать, чтоб пользователь видел только таблицу внутри цикла, а таблицу цикла не видел?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
В Ролях параметр Таблица с одним циклом — на основе нее ЛК обычно делаются.

Не забудте поставить для таблицы циклов ограничение Доступ только для владельца цикла!

Если цикла для пользователя нет, то он будет создан при переходе в табл циклов с одним ц.

**Reply by aityomin** [reply]
Спасибо!
